src/mockserver.js

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs';
+
+import { logger } from '@user-office-software/duo-logger';
+import { mockServerClient } from 'mockserver-client';
+
+async function mockserver() {
+    var callback = function (request) {
+        if (request.method === 'POST') {
+            const name = String(request.body.xml);
+            let regexp = '<tns:(.*?)>';
+            const test = name.match(regexp);
+            const method = test[1];
+
+            if (
+                name.includes('getBasicPersonDetailsFromUserNumber') ||
+                name.includes('getSearchableBasicPeopleDetailsFromUserNumbers')
```

Comment:
In many of these `name.includes()` checks (the ones later on too), it looks like we're only cheking the method name. Can we change them to `method === <method name>` instead, to make this clearer? It should also be a bit faster to run